SPM Example/PortalSwift/ViewController.swift

```diff
@@ -1376,4 +1376,44 @@ class ViewController: UIViewController, UITextFieldDelegate {
       }
     }
   }
+  
+  func handleSwaps() {
+    do {
+      self.startLoading()
+
+      guard let portal else {
+        self.logger.error("ViewController.handleSwaps() - ❌ Portal not initialized")
+        throw PortalExampleAppError.portalNotInitialized()
+      }
+      
+      let swaps = PortalSwaps(apiKey: "6b634597-d4fc-4001-95e6-541de1d69fe8", portal: portal)
+      
+      swaps.getSources() { result in
+        let source = result
+        print("getSources response:", source)
+
+        // Delay the second request by 1 second
+        DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
+          let quoteArgs = QuoteArgs(
+            buyToken: "0x68194a729c2450ad26072b3d33adacbcef39d574", // USDC on Sepolia
+            sellToken: "ETH",
+            sellAmount: "1"
+          )
+          
+          let customChainId = "eip155:11155111"
+          swaps.getQuote(args: quoteArgs, forChainId: customChainId) { result in
```

Comment:
should we also test sending the quote after we get it from 0x? i understand there is a liquidity issue so we might not be able to do that anyways.

Reply:
yeah we can't do that now but that would be wise to do

Reply:
Does this method still work if you dont include forChainId?

Reply:
ah gotcha from youre response below